src/bun.js/javascript.zig

```diff
@@ -760,11 +761,22 @@ pub const VirtualMachine = struct {
             this.hide_bun_stackframes = false;
         }
 
-        if (map.map.fetchSwapRemove("BUN_INTERNAL_IPC_FD")) |kv| {
-            if (std.fmt.parseInt(i32, kv.value.value, 10) catch null) |fd| {
-                this.initIPCInstance(bun.toFD(fd));
+        if (map.map.fetchSwapRemove("BUN_INTERNAL_IPC_PIPE")) |kv| {
```

Comment:
is there a reason we cant use the same environment variable as it is on posix.

one lookup instead of two?